### PR TITLE
Create Accessibility Avery

### PR DESCRIPTION
**Pull request by mikang**

### CODE REVIEW

**Review by dmundra — CHANGES_REQUESTED** [review]
@mikang missing .md for and headers for this one as well. Similar to #237

**Comment by mikang** [comment]
I've made the changes @dmundra - does it notify you now to merge? Is there another action I should take?

File: Personas/accessibility-avery.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Accessibility Avery
+
+# Description
```
Comment (dmundra) [inline Personas/accessibility-avery.md:3]: @mikang I think these and below headers should be second-level so `##`

**Comment by dmundra** [comment]
> I've made the changes @dmundra - does it notify you now to merge? Is there another action I should take?

@mikang it does. I did notice a minor issue with headers. I think the rest of the headers after the first one should be second-level.

**Comment by mikang** [comment]
@dmundra nice! both on the notifications and the change request. I've made the update with the ##s.

**Review by dmundra — APPROVED** [review]

**Comment by dmundra** [comment]
Looking good @mikang. Pa11y-ci had a crash, you can just manually merge it.

**Pull request merged**